liquid_node/commands.py

```diff
@@ -217,6 +219,13 @@ def start(job, hcl):
         job_checks = start(job, hcl)
         health_checks.update(job_checks)
 
+    # Wait for database health checks of all collections and hoover:
+    pg_checks = {k: v for k, v in health_checks.items() if k.endswith("-pg")}
```

Comment:
This feels dangerous, we've used the `-pg` convention so far, and will probably do so in the future, but I'd rather flag those tasks explicitly. Maybe add a `pg_tasks` list in [liquid_node.jobs.hoover.Hoover](https://github.com/liquidinvestigations/node/blob/90afe87bc65ef4a298b409a65b9d67cb466d60e4/liquid_node/jobs/hoover.py#L4)?

Reply:
That would imply doing what @gabriel-v suggested and putting the db tasks into separate jobs?! I think that it would clutter the interface. We already have 2 jobs per collection and we have to click what feels like 20 times to see logs. 
For that reason, I'd rather keep the db as a task. What does you guys think? 
I could also offer:
`pg_checks = {k: v for k, v in health_checks.items() if k.endswith("-pg") and (k.startswith("snoop") or k.startswith("hoover"))}`   😀

Reply:
Yes, I think you should keep the `-pg` tasks as they are. Just don't have any `.endswith` and `.startswith` logic, just re-build a list of all your checks on the fly (the list is `config.collections`)